src/core/Akka.API.Tests/CoreAPISpec.ApprovePersistence.approved.txt

```diff
@@ -243,6 +243,7 @@ namespace Akka.Persistence
         public void PersistAsync<TEvent>(TEvent @event, System.Action<TEvent> handler) { }
         protected abstract bool ReceiveCommand(object message);
         protected abstract bool ReceiveRecover(object message);
+        protected void RunTask(System.Func<System.Threading.Tasks.Task> action) { }
```

Comment:
This method is not unique for the persistence. It should be either hidden or included in `ActorBase` class.